client/src/client-only-routes/show-certification.tsx

```diff
@@ -235,6 +235,11 @@ const ShowCertification = (props: ShowCertificationProps): JSX.Element => {
   const certMonth = certDate.getMonth();
   const certURL = `https://freecodecamp.org${pathname}`;
 
+  const currentTheme =
+    localStorage.getItem('theme') == Themes.Night
+      ? Themes.Night
+      : Themes.Default;
+
```

Comment:
```suggestion
```

The theme on the certification page is always default.

Reply:
I suggest making a function called getThemeClass and move all of the related logic inside.
This also needs to set the Theme to localStorage if not found in localStronge.
  
Also the theme names need to be standardized to 'light' and 'dark' when they are saved.
That will help us carry the same naming convention from the ui component library and our variables to the theme value storage.

Reply:
I have a new issue. Unfortunately for me, the theme toggle is behaving very oddly. It isn't updating automatically any more.